### DISCUSSION TITLE
Include sub properties inside Component-api  docs element search results

### DISCUSSION THREAD

**Discussion by vandelpavel** [quasarframework/quasar#15641] [Ideas / Proposals]
## Intro
When you search for a props/methods/events/ecc... you will see the results in the below section like in this snapshot:

![image](https://user-images.githubusercontent.com/61196133/228578234-f96ce3d9-d741-4fda-a63d-8e08e2a43976.png)

But for example if you try to search for events returned parameters you wouldn't be able to see it in the result.

# Repro case:
Search inside the **Q-Toggle** API the **evt** property returned from the @update:model-value event

# Expected behaviour:
 You should see the property in the list

![image](https://user-images.githubusercontent.com/61196133/228577112-d1fd8275-2fc2-4ce0-a823-750621c8fd2d.png)

# Current Behaviour:
 The event containing the property isn't shown

![image](https://user-images.githubusercontent.com/61196133/228577994-e0a4fb17-7d89-4835-85ed-f89e7e7c93ad.png)

**Comment by rstoenescu** [comment]
This would indeed be nice. Adding it to the TODO list.
